docs/luigi-core-api.md

```diff
@@ -190,7 +190,7 @@ Luigi.readUserSettings();
 
 #### storeUserSettings
 
-Writes the user settings object.
+Reads the user settings object.
```

Comment:
I'm not so sure why this was changed. Was it not supposed to be _'writes'_ ?

Reply:
mmm yes sure.... I am wondering y I changed this field!!!